remoting/src/main/java/org/apache/rocketmq/remoting/netty/NettyClientConfig.java

```diff
@@ -20,23 +20,23 @@
     /**
      * Worker thread number
      */
-    private int clientWorkerThreads = 4;
+    private int clientWorkerThreads = NettySystemConfig.clientWorkerSize;
     private int clientCallbackExecutorThreads = Runtime.getRuntime().availableProcessors();
     private int clientOnewaySemaphoreValue = NettySystemConfig.CLIENT_ONEWAY_SEMAPHORE_VALUE;
     private int clientAsyncSemaphoreValue = NettySystemConfig.CLIENT_ASYNC_SEMAPHORE_VALUE;
-    private int connectTimeoutMillis = 3000;
+    private int connectTimeoutMillis = NettySystemConfig.connectTimeoutMillis;
     private long channelNotActiveInterval = 1000 * 60;
```

Comment:
Is this param should be parameterized？

Reply:
i think this value may be different for different network situations